src/docs/auth/README.md

```diff
@@ -19,3 +19,25 @@ No es un documento sino que una página web.
 - MD5: `05906d13a4532dd32f12d260fd0a88d5`
 - SHA256: `3cf33550b6eb5a448d5b4c48063f8d74a48b1435b2671675929d981a1c1ec03a`
 - Fecha y hora descarga: 2020-02-07 16:27 -03:00
+
+
+## Tech
+
+
+### Web services "Autenticación Automática con Certificado Digital"
+
+(`CrSeed`) y (`GetTokenFromSeed`)
+
+"Manual de Desarrollador" - "Autenticación Automática",
+[v1.9 (2007-11-18)](wsAutenticacionAutomatica/2007-11-18-autenticacion-automatica-v1.9.pdf)
+
+- Fuente: 
+  [Sitio Web SII](http://homer.sii.cl/) 
+  / [RTC](https://palena.sii.cl/rtc/RTC/RTCMenu.html)
+  / Documentación Técnica
+  / [Automatización de Procesos](https://palena.sii.cl/rtc/RTC/RTCDocAutom.html)
+  / [Webservice Autenticación Automática con Certificado Digital](https://palena.sii.cl/cgi_rtc/RTC/RTCDocum.cgi?1)
+  (**sólo disponible con autenticación con certificado digital**).
+- Nombre original: `autenticacion.pdf`
+- MD5: `73cb767473d57d227da424928365abe4`
+- SHA256: `2193c9cabba2a3e9a01774eea6afdd21f7071be27709d67e513e475ebb386ecc`
```

Comment:
Please add date and time of file retrieval. See https://github.com/cl-sii-extraoficial/archivos-oficiales/pull/7/files#diff-dec72b4b8f66620caa27cf3945bcd75357ccd5f263e31bf34dbeface2fe2c7d3R21 for an example.

Reply:
Since I did not consider it the first time I downloaded it, I download it again (luckily the document has not changed 😅 )